src/cmds/scripts/pbs_postinstall.in

```diff
@@ -469,6 +469,11 @@ elif [ $INSTALL_PACKAGE != client ] ; then
 			[ -f /etc/profile.d/pbs.csh ] || cp ${PBS_EXEC}/etc/pbs.csh /etc/profile.d
 			[ -f /etc/profile.d/pbs.sh ] || cp ${PBS_EXEC}/etc/pbs.sh /etc/profile.d
 		fi
+		unitfilepath="/usr/lib/systemd/system/"
+		if [ -d $unitfilepath ]; then
+			cp ${PBS_EXEC}/libexec/pbs.service $unitfilepath
```

Comment:
Also, the documented standard location of the unit file installed through RPM is /usr/lib/systemd/system/. 
https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/chap-Managing_Services_with_systemd.html#tabl-Managing_Services_with_systemd-Introduction-Units-Locations

This exactly is the location that you are using. so better use it in the "standard" way.

I assume that the reason behind doing the way you have done is that we are not sure if the folder " /usr/lib/systemd/system/ " exists.

There is another way of doing this using RPM macros in the spec file which will need you to do more exploration of distribution specific ways of finding the version of the OS. 
For Redhat, information is at -
https://fedoraproject.org/wiki/Packaging:DistTag
For SLES - 
https://en.opensuse.org/openSUSE:Systemd_packaging_guidelines.